rasa/nlu/extractors/duckling_entity_extractor.py

```diff
@@ -51,50 +58,48 @@ def convert_duckling_format_to_rasa(
     return extracted
 
 
-class DucklingEntityExtractor(EntityExtractor):
-    """Searches for structured entites, e.g. dates, using a duckling server."""
-
-    defaults = {
-        # by default all dimensions recognized by duckling are returned
-        # dimensions can be configured to contain an array of strings
-        # with the names of the dimensions to filter for
-        "dimensions": None,
-        # http url of the running duckling server
-        "url": None,
-        # locale - if not set, we will use the language of the model
-        "locale": None,
-        # timezone like Europe/Berlin
-        # if not set the default timezone of Duckling is going to be used
-        "timezone": None,
-        # Timeout for receiving response from http url of the running duckling server
-        # if not set the default timeout of duckling http url is set to 3 seconds.
-        "timeout": 3,
-    }
-
-    def __init__(
-        self,
-        component_config: Optional[Dict[Text, Any]] = None,
-        language: Optional[Text] = None,
-    ) -> None:
-
-        super().__init__(component_config)
-        self.language = language
+class DucklingEntityExtractorComponent(GraphComponent, EntityExtractorMixin):
+    """Searches for structured entities, e.g. dates, using a duckling server."""
 
-    @classmethod
-    def create(
-        cls, component_config: Dict[Text, Any], config: RasaNLUModelConfig
-    ) -> "DucklingEntityExtractor":
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        """The component's default config (see parent class for full docstring)."""
+        return {
+            # by default all dimensions recognized by duckling are returned
+            # dimensions can be configured to contain an array of strings
+            # with the names of the dimensions to filter for
+            "dimensions": None,
+            # http url of the running duckling server
+            "url": None,
+            # locale - if not set, we will use the language of the model
+            "locale": None,
+            # timezone like Europe/Berlin
+            # if not set the default timezone of Duckling is going to be used
+            "timezone": None,
+            # Timeout for receiving response from HTTP URL of the running
+            # duckling server. If not set the default timeout of duckling HTTP URL
+            # is set to 3 seconds.
+            "timeout": 3,
+        }
 
-        return cls(component_config, config.language)
```

Comment:
Should `language` also be added to the component `config`?

Reply:
I'd just put that into the `locale` (see my comment [here](https://github.com/RasaHQ/rasa/pull/9564#discussion_r703594270))